RELEASE.md

```diff
@@ -27,20 +27,24 @@ work. Here's one example workflow (assuming of course you have rights to
 directly push to the upstream repo).
 
 Create a temporary clone of the repo, e.g. to `/tmp`. Then create the `tag` and
-push to origin. For example, to create a release called `v0.24.0`, you would run
-the following commands:
+push to origin. Since knative 1.0 the releases in this repo still are versioned in the `v0.x.y` format, and the tags corresponds to the knative tags `knative-v1.x.y`. This starts from `v0.27.0` == `knative-v1.0.0` and so on. For example, to create a release called `v0.27.0`, you would run the following commands:
```

Comment:
Just to verify: if I create tag `v0.28.0` that would correspond to `knative-v1.1.0` then?

Reply:
Perhaps another example (briefly) to describe the mapping for when knative-1.1.0 is to be created would be neat.